src/less/variables.less

```diff
@@ -175,10 +181,12 @@
 
 // Buttons
 //-------------------------
-@btn-default-color:           #fff;
-@btn-default-bg:              @gray-400;
+@btn-default-color:           @black;
+@btn-default-bg:              @body-bg;
+@btn-default-border:          @gray-150;
+
 
-@btn-primary-color:           @btn-default-color;
+@btn-primary-color:           @body-bg;
```

Comment:
Added btn-default-color, but the current value is #fff, should we keep it like that or does it need a different value?